plinth/modules/shaarli/views.py

```diff
@@ -28,6 +28,9 @@
 from plinth import package
 from plinth.modules import shaarli
 
+def on_install():
+"""Notify that the service is now enabled."""
+shaarli.service.notify_enabled(None, True)
 
 @package.required(['shaarli'])
```

Comment:
One more modification is needed. Line 35 should be:
```
@package.required(['shaarli'], on_install=on_install)
```
The first "on_install" is an optional parameter for the @package.required function. And the 2nd "on_install" is a reference to the function that you defined above.